RealmLoginKit Android/app/src/main/java/io/realm/realmloginkit/example/ExampleActivity.java

```diff
@@ -0,0 +1,168 @@
+        initTheme(); // it should be invoked before setContentView()
+        setContentView(R.layout.activity_example);
+        initThemeButtons(isDarkMode);
+        initLogo();
+        findViewById(R.id.log_in).setOnClickListener(this);
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        delayedResizeLogo();
+    }
+
+    @Override
+    public void onClick(View view) {
+        final int viewId = view.getId();
+        handleThemeIfNeeded(viewId);
+        handleLogInIfNeeded(viewId);
+    }
+
+    private void initLogo() {
+        logo = (RealmLogoView) findViewById(R.id.logo);
+        logo.setMonochromeLogo(isDarkMode);
+    }
+
+    private void delayedResizeLogo() {
+        new Handler().postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                resizeLogo();
+            }
+        }, 100);
+    }
+
+    private void resizeLogo() {
+        final int width = getWindow().getDecorView().getWidth();
+        final int height = getWindow().getDecorView().getHeight();
+        final int minWidth;
+        if (width < height) {
+            minWidth = (int) (width * LOGO_RATIO);
+        } else {
+            minWidth = (int) (height * LOGO_RATIO);
+        }
+        ViewGroup.LayoutParams layoutParams = logo.getLayoutParams();
+        layoutParams.width = minWidth;
+        layoutParams.height = minWidth;
+        logo.setLayoutParams(layoutParams);
+    }
+
+    private void initThemeButtons(boolean isDarkMode) {
+        final ActionBar actionBar = getSupportActionBar();
+        actionBar.setCustomView(R.layout.theme_toggles);
+        actionBar.setDisplayOptions(ActionBar.DISPLAY_SHOW_CUSTOM);
+        final View customView = actionBar.getCustomView();
+
+        lightButton = (ToggleButton) customView.findViewById(R.id.light_button);
+        lightButton.setOnClickListener(this);
+
+        darkButton = (ToggleButton) customView.findViewById(R.id.dark_button);
+        darkButton.setOnClickListener(this);
+
+        if (isDarkMode) {
+            darkButton.setChecked(true);
+            lightButton.setChecked(false);
+        } else {
+            lightButton.setChecked(true);
+            darkButton.setChecked(false);
+        }
+    }
+
+    private void initTheme() {
+        if (isDarkMode) {
+            setTheme(android.support.v7.appcompat.R.style.Theme_AppCompat);
+        } else {
+            setTheme(android.support.v7.appcompat.R.style.Theme_AppCompat_Light);
+        }
+    }
+
+    private void handleLogInIfNeeded(int viewId) {
+        if (viewId == R.id.log_in) {
+            LoginKit.loginKit(this)
+                    .setDarkMode(isDarkMode)
+                    .setAppTitle("Example App")
+                    .setServerUri("127.0.0.7", false)
+                    .logIn();
+        }
+    }
+
+    private void handleThemeIfNeeded(int viewId) {
+        if (viewId == R.id.light_button) {
```

Comment:
Perhaps this should be a switch?  If R.id.light_button is a constant (this is not a library) that will work.